CMSgov/easi-app
# Program Background documentation

## Changes and Description

Documentation add around team composition.



## How to test this change

Read the new documentation and make sure that you can navigate to it from the main readme page.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
